lib/core/public/run.js

```diff
@@ -36,10 +36,10 @@ export default function run(...args) {
     axe.utils.performanceTimer.start();
   }
 
-  function handleRunRules(rawResults, cleanup) {
+  function handleRunRules(rawResults, teardown) {
```

Comment:
Since this is actually just the teardown method, it's confusing to have it named something else here.